lib/config/rules/best-practices.js

```diff
@@ -138,6 +140,8 @@ module.exports = {
   radix: 'error',
   // Disallow async functions which have no await expression
   'require-await': 'error',
+  // Enforce the use of u flag on RegExp <FEEDBACK>
+  'require-unicode-regexp': 'off',
```

Comment:
had this enabled originally but we offend it so much that we'll likely just turn it off.